coalahtml/helper.py

```diff
@@ -7,6 +8,43 @@
 from coalahtml.tree import Tree
 
 
+class DirCmp(filecmp.dircmp):
+    """
+    Compare the content of dir1 and dir2. In contrast with filecmp.dircmp,
+    this subclass compares the content of files with the same path.
+    """
+
+    def phase3(self):
+        """
+        Find out differences between common files.
+        Ensure we are using content comparison with shallow=False.
+        Ref: https://hg.python.org/cpython/file/2.7/Lib/filecmp.py#l174
```

Comment:
Referencing to Python 2.7 doesn't seem right, cause you're not using that version. Also, check if this doesn't already exist in Python 3